docs/using.rst

```diff
@@ -50,8 +114,128 @@ The ``letsencrypt`` commandline tool has a builtin help:
    Let's Encrypt team!
 
 
+Plugins
+=======
+
+Officially supported plugins:
+
+========== = = ================================================================
+Plugin     A I Notes and status
+========== = = ================================================================
+standalone Y N Very stable. Uses port 80 (force by
+               ``--standalone-supported-challenges simpleHttp``) or 443
+               (force by ``standalone-supported-challenges dvsni``).
+webroot    Y N Works with already running webserver, by writing necessary files
+               to the disk (``--webroot-path`` should be pointed to your
+               ``public_html``). Currently, when multiple domains are specified
+               (`-d`), they must all use the same web root path.
+manual     Y N Hidden from standard UI, use with ``--a manual``. Requires to
+               copy and paste commands into a new terminal session. Allows to
+               run client on machine different than target webserver, e.g. your
+               laptop.
+apache     Y Y Alpha - might break stuff, so be careful. Support for
```

Comment:
"might break stuff, so be careful" -> "automates Apache installation, works fairly well but Debian only".  Move Apache up to position 2 I think.